tapeout/src/main/scala/transforms/Generate.scala

```diff
@@ -99,7 +147,26 @@ sealed trait GenerateTopAndHarnessApp extends LazyLogging { this: App =>
       customTransforms = firrtlOptions.customTransforms ++ topTransforms
     )
 
-    firrtl.Driver.execute(optionsManager)
+    val result = firrtl.Driver.execute(optionsManager)
+
+    result match {
```

Comment:
Do you care about what level of firrtl is emitted? This will be the lowest level but you could also do the emission after the last custom transform to get a higher level of firrtl.

Reply:
I think the lowest level is probably fine here, unless you see a reason to produce a higher form.